app/main.go

```diff
@@ -62,8 +68,24 @@ func getIPAddress(r *http.Request) string {
 }
 
 func handlerPing(w http.ResponseWriter, r *http.Request) {
-	w.Write([]byte("pong"))
-	fmt.Println(time.Now(), getIPAddress(r), r.Method, r.RequestURI, r.UserAgent())
+	
+	requestBody, err := ioutil.ReadAll(r.Body)
+	n := rand.Intn(100)
+
+	<- time.After(time.Duration(lat) * time.Millisecond)
+	if n <= prob {
+		w.Write([]byte("pong"))
+		fmt.Println(time.Now(), getIPAddress(r), r.Method, r.RequestURI, r.UserAgent())
```

Comment:
We could just print once per request. Ideally we have only single log line per request (if any).

The point of metrics is to not use logging at all even, because we have all data we want (metric per each request)

Reply:
Ok, let's remove all logging then :)